memq-actor/src/main/java/io/appform/memq/actor/HighLevelActorConfig.java

```diff
@@ -12,25 +12,26 @@
 import javax.validation.constraints.NotNull;
 
 @Data
-@EqualsAndHashCode
+@Builder
 @ToString
+@EqualsAndHashCode
 @AllArgsConstructor
 @NoArgsConstructor
-@Builder
-public class ActorConfig {
+public class HighLevelActorConfig {
 
     @Min(1)
     @Max(100)
     @Builder.Default
     private int partitions = 1;
 
-    @NotNull
     @Valid
+    @NotNull
     @Builder.Default
     private RetryConfig retryConfig = new NoRetryConfig();
 
-    @NotNull
     @Valid
+    @NotNull
+    @Builder.Default
     private ExceptionHandlerConfig exceptionHandlerConfig = new SidelineConfig();
```

Comment:
This should be drop by default. Because we do not really have a decent sidelining system by default

Reply:
Done